backend/coins/eth/account.go

```diff
@@ -459,6 +459,8 @@ func (account *Account) newTx(args *accounts.TxProposalArgs) (*TxProposal, error
 		return nil, errp.WithStack(errors.ErrFeesNotAvailable)
 	}
 
+	account.Synchronizer.WaitSynchronized()
```

Comment:
Would've been nice to have a comment like this:

```go
// Make sure account.balance is up to date for calculations below.
account.Synchronizer.WaitSynchronized()
```

Reply:
Added :+1: